index2.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style2.css">
+
+    <title>Document</title>
+</head>
+<body>
+   <div class="header">
+    <img class="cell-info" src="Light.png"></div>
+    <div class="bottom-section">
+    <p class="welcome">Welcome</p>
+    <img class="union" src="Union.png">
+    </div>
+   </div>
```

Comment:
- [ ] In order to make your HTML semantic and organized, please use `<header>` tag. Div is not prefferable when you are creating a wrapper such as header, main, article, section, footer, form etc... If you can't find a semantic tag for your purpose, you can use `div` then. Also for `img` tags, kindly add alt attribute to make it more accessible by screen-readers. You can define the purpose of the image inside it. Such as logo, menuBtn... and please don't forget to add relative path in your src attributes like this `./Union.png` . Otherwise, it's not gonna show off in other browsers.